pkg/kyverno/apply/apply_command.go

```diff
@@ -294,6 +294,7 @@ func applyCommandHelper(resourcePaths []string, cluster bool, policyReport bool,
 				}
 			}
 		}
+		globalValMap = common.Set_default_value_request_operation_to_create(matches, globalValMap)
```

Comment:
Why are we updating the globalVarMap in a for loop? This means that the order of policies will cause different outcomes for the globalVarMap. We should only be defining this once closer to the declaration of globalVarMap